xarray/coding/cftime_offsets.py

```diff
@@ -1218,12 +1269,154 @@ def date_range(
         If True, always return a CFTimeIndex.
         If False, return a pd.DatetimeIndex if possible or raise a ValueError.
         If None (default), return a pd.DatetimeIndex if possible,
-        otherwise return a CFTimeIndex. Defaults to False if `tz` is not None.
+        otherwise return a CFTimeIndex. Overridden to False if `tz` is not None.
 
     Returns
     -------
     CFTimeIndex or pd.DatetimeIndex
 
+    Notes
+    -----
+    When use_cftime=True, this function is an analog of ``pandas.date_range``
```

Comment:
This is great, thanks!  One minor other detail here:
```suggestion
    When ``use_cftime=True``, or a calendar other than "standard", "gregorian",
    or "proleptic_gregorian" is provided, this function is an analog of ``pandas.date_range``
```